clippy_lints/src/collapsible_if.rs

```diff
@@ -163,6 +164,54 @@ impl CollapsibleIf {
         }
     }
 
+    fn check_collapsible_if_nested_if_else(context: &LateContext<'_>, if_expr: &Expr<'_>) {
```

Comment:
Please use the same names as throughout the rest of Clippy:

```suggestion
    fn check_collapsible_if_nested_if_else(cx: &LateContext<'_>, if_expr: &Expr<'_>) {
```

Also, if this doesn't require a `self`, you should get it out of the `CollapsibleIf` struct impl.